### ISSUE TITLE
Integration Testing, Deployment and Fixes for Reporting features

### ISSUE THREAD

**Issue by mpilchar** [bcgov/foi-flow#1308]
Title of ticket: 

#### Description
Summarize issue 

#### Dependencies
Are there any dependencies?

#### DOD
- [ ] List the items that need to be complete for this ticket to be considered done
- [ ] 
- [ ] 
- [ ] 
- [ ]

**Comment by mpilchar** [comment]
will close once #1406 is complete

**Comment by mpilchar** [comment]
New ticket being created to capture feedback from Andrew
